cromwell_tools/cromwell_tools.py

```diff
@@ -114,6 +115,7 @@ def wait_until_workflow_completes(
             time.sleep(poll_interval_seconds)
 
 
+@retry(wait=wait_exponential(multiplier=1, max=10), stop=stop_after_delay(20))
```

Comment:
It would be really great if you could update the docstring here to reflect the new attribute `.retry.statistics` which is added to this function. Then we could use something like `logging.info(cromwell_tools.start_workflow.retry.statistics)` in lira or other places

Reply:
Great idea!